!SESSION 2024-07-31 17:55:04.741 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-07-31 17:57:13.798
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more

!ENTRY org.eclipse.cdt.core 1 0 2024-07-31 17:59:49.162
!MESSAGE Indexed 'hello_lab1_bsp' (89 sources, 68 headers) in 2.48 sec: 1,444 declarations; 3,713 references; 125 unresolved inclusions; 6 syntax errors; 212 unresolved names (3.9%)

!ENTRY org.eclipse.cdt.core 1 0 2024-07-31 17:59:56.374
!MESSAGE Indexed 'hello_lab1' (1 sources, 0 headers) in 0.006 sec: 1 declarations; 1 references; 1 unresolved inclusions; 0 syntax errors; 0 unresolved names (0%)
!SESSION 2024-08-08 14:13:47.204 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-08-08 14:15:36.029
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more
!SESSION 2024-08-08 14:31:34.101 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.core.resources 2 10035 2024-08-08 14:31:37.790
!MESSAGE The workspace exited with unsaved changes in the previous session; refreshing workspace to recover changes.

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-08-08 14:31:48.626
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more

!ENTRY org.eclipse.cdt.core 1 0 2024-08-08 14:51:21.876
!MESSAGE Indexed 'lab2_bsp' (89 sources, 160 headers) in 2.09 sec: 5,791 declarations; 9,745 references; 39 unresolved inclusions; 3 syntax errors; 81 unresolved names (0.52%)

!ENTRY org.eclipse.cdt.core 1 0 2024-08-08 14:51:22.595
!MESSAGE Indexed 'lab2' (1 sources, 0 headers) in 0.004 sec: 1 declarations; 1 references; 1 unresolved inclusions; 0 syntax errors; 0 unresolved names (0%)
!SESSION 2024-08-08 14:56:28.347 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-08-08 14:56:54.309
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: dev.eclipse.org. ; version: 1.100.0.v20160217-0435
!STACK 0
java.net.UnknownHostException: dev.eclipse.org
	at java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.net.InetAddress$2.lookupAllHostAddr(Unknown Source)
	at java.net.InetAddress.getAddressesFromNameService(Unknown Source)
	at java.net.InetAddress.getAllByName0(Unknown Source)
	at java.net.InetAddress.getAllByName(Unknown Source)
	at java.net.InetAddress.getAllByName(Unknown Source)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.HttpClientConnectionOperator.connect(HttpClientConnectionOperator.java:102)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:319)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:363)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:219)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:195)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:86)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
	at org.apache.http.client.fluent.Executor.execute(Executor.java:206)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.request(IO.java:205)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:68)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 14:58:50.559
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 14:58:57.650
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 14:58:57.676
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 14:59:03.671
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --go, /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf] return code: 4, 'other error'

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 14:59:03.674
!MESSAGE Downloading ELF Process failed

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:04:09.587
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop]
!SESSION 2024-08-08 15:06:53.684 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.core.resources 2 10035 2024-08-08 15:06:58.779
!MESSAGE The workspace exited with unsaved changes in the previous session; refreshing workspace to recover changes.

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-08-08 15:07:10.213
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:08:07.316
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 15:08:11.237
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:08:11.264
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:10:55.336
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:10:55.738
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 15:10:57.868
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:10:57.885
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:12:07.158
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:12:07.532
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 15:12:09.657
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:12:09.673
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:17:20.143
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:17:20.523
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 15:17:22.637
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:17:22.652
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:18:47.193
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:18:47.600
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 15:18:49.770
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:18:49.786
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:20:17.612
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:20:17.977
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 15:20:20.046
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:20:20.062
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:22:47.024
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:22:47.396
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 15:22:49.483
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:22:49.500
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:23:38.674
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:23:39.034
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 15:23:41.115
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:23:41.130
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:26:17.378
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:26:17.769
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 15:26:19.894
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:26:19.909
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:49:23.016
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:49:23.388
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 15:49:25.473
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:49:25.489
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:57:21.544
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:57:21.909
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-08 15:57:24.011
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-08 15:57:24.026
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]
!SESSION 2024-08-15 14:14:29.265 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-08-15 14:20:55.868
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:22:23.303
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:22:29.640
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:22:29.668
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:30:24.230
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:30:24.628
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:30:26.804
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:30:26.821
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:30:49.698
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:30:50.115
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:30:52.371
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:30:52.389
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:31:10.615
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:31:10.993
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:31:13.093
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:31:13.109
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:31:39.455
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:31:39.842
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:31:41.963
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:31:41.980
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:33:47.789
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:33:48.163
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:33:50.267
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:33:50.285
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:34:58.321
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:34:58.732
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:35:00.916
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:35:00.932
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:37:59.099
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:37:59.530
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:38:01.679
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:38:01.696
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:40:14.088
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:40:14.508
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:40:16.606
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:40:16.623
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:41:25.026
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:41:25.457
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:41:27.657
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:41:27.674
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:48:16.993
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:48:17.377
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:48:19.496
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:48:19.514
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:50:50.994
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:50:51.407
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:50:53.514
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:50:53.529
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:51:52.569
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:51:52.946
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:51:55.041
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:51:55.057
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:54:31.112
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:54:31.550
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 14:54:33.677
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 14:54:33.693
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:08:33.869
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:08:34.257
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:08:36.389
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:08:36.405
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:10:30.416
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:10:30.885
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:10:33.048
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:10:33.064
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:11:18.977
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:11:19.417
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:11:21.645
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:11:21.667
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:12:42.606
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:12:43.011
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:12:45.111
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:12:45.127
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:14:10.469
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:14:10.906
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:14:13.144
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:14:13.163
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:15:22.270
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:15:22.659
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:15:24.737
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:15:24.756
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:17:35.308
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:17:35.751
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:17:37.989
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:17:38.013
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:19:31.524
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:19:31.948
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:19:34.056
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:19:34.072
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:21:15.117
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:21:15.533
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:21:17.666
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:21:17.684
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:22:54.544
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:22:54.986
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:22:57.133
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:22:57.153
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:23:23.175
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:23:23.675
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:23:25.829
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:23:25.968
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:25:51.513
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:25:51.982
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:25:54.143
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:25:54.169
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:27:44.560
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:27:44.983
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --accept-bad-sysid --sidp=Not Found]

!ENTRY com.altera.sbtgui.launch 4 0 2024-08-15 15:27:47.139
!MESSAGE Failed Executing: [nios2-download, '--cable=USB-Blaster on localhost [USB-0]', --device=1, --instance=0, --stop, --accept-bad-sysid, --sidp=Not Found] return code: 8

!ENTRY com.altera.sbtgui.launch 1 0 2024-08-15 15:27:47.158
!MESSAGE Executing: [D:/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --accept-bad-sysid /cygdrive/c/Users/lachl/OneDrive/Desktop/2024/COMPSYS301/labs/software/lab2/lab2.elf]
!SESSION 2024-08-15 15:57:28.699 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-08-15 15:57:58.402
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more
!SESSION 2024-09-02 16:05:55.880 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-09-02 16:06:25.772
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more
!SESSION 2024-09-19 14:02:41.635 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-09-19 14:42:00.069
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more
!SESSION 2024-09-19 15:43:09.786 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_NZ
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/intel/fpga/18.10/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/intel/fpga/18.10/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.core.resources 4 567 2024-09-19 15:43:36.827
!MESSAGE Workspace restored, but some problems occurred.
!SUBENTRY 1 org.eclipse.core.resources 4 567 2024-09-19 15:43:36.827
!MESSAGE Could not read metadata for 'hello_lab1'.
!STACK 1
org.eclipse.core.internal.resources.ResourceException: The project description file (.project) for 'hello_lab1' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.read(FileSystemResourceManager.java:888)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:902)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:882)
	at org.eclipse.core.internal.resources.SaveManager.restore(SaveManager.java:733)
	at org.eclipse.core.internal.resources.SaveManager.startup(SaveManager.java:1588)
	at org.eclipse.core.internal.resources.Workspace.startup(Workspace.java:2386)
	at org.eclipse.core.internal.resources.Workspace.open(Workspace.java:2157)
	at org.eclipse.core.resources.ResourcesPlugin.start(ResourcesPlugin.java:463)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:771)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:764)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:721)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:941)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:318)
	at org.eclipse.osgi.container.Module.doStart(Module.java:571)
	at org.eclipse.osgi.container.Module.start(Module.java:439)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:454)
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:107)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:531)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:324)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:327)
	at org.eclipse.osgi.internal.loader.sources.SingleSourcePackage.loadClass(SingleSourcePackage.java:36)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:398)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:352)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:344)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:160)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:140)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:380)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:235)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:669)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:608)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1515)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1488)
!SUBENTRY 2 org.eclipse.core.resources 4 567 2024-09-19 15:43:36.827
!MESSAGE The project description file (.project) for 'hello_lab1' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
!SUBENTRY 1 org.eclipse.core.resources 4 567 2024-09-19 15:43:36.827
!MESSAGE Could not read metadata for 'hello_lab1_bsp'.
!STACK 1
org.eclipse.core.internal.resources.ResourceException: The project description file (.project) for 'hello_lab1_bsp' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.read(FileSystemResourceManager.java:888)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:902)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:882)
	at org.eclipse.core.internal.resources.SaveManager.restore(SaveManager.java:733)
	at org.eclipse.core.internal.resources.SaveManager.startup(SaveManager.java:1588)
	at org.eclipse.core.internal.resources.Workspace.startup(Workspace.java:2386)
	at org.eclipse.core.internal.resources.Workspace.open(Workspace.java:2157)
	at org.eclipse.core.resources.ResourcesPlugin.start(ResourcesPlugin.java:463)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:771)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:764)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:721)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:941)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:318)
	at org.eclipse.osgi.container.Module.doStart(Module.java:571)
	at org.eclipse.osgi.container.Module.start(Module.java:439)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:454)
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:107)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:531)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:324)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:327)
	at org.eclipse.osgi.internal.loader.sources.SingleSourcePackage.loadClass(SingleSourcePackage.java:36)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:398)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:352)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:344)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:160)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:140)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:380)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:235)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:669)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:608)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1515)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1488)
!SUBENTRY 2 org.eclipse.core.resources 4 567 2024-09-19 15:43:36.828
!MESSAGE The project description file (.project) for 'hello_lab1_bsp' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
!SUBENTRY 1 org.eclipse.core.resources 4 567 2024-09-19 15:43:36.828
!MESSAGE Could not read metadata for 'lab2'.
!STACK 1
org.eclipse.core.internal.resources.ResourceException: The project description file (.project) for 'lab2' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.read(FileSystemResourceManager.java:888)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:902)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:882)
	at org.eclipse.core.internal.resources.SaveManager.restore(SaveManager.java:733)
	at org.eclipse.core.internal.resources.SaveManager.startup(SaveManager.java:1588)
	at org.eclipse.core.internal.resources.Workspace.startup(Workspace.java:2386)
	at org.eclipse.core.internal.resources.Workspace.open(Workspace.java:2157)
	at org.eclipse.core.resources.ResourcesPlugin.start(ResourcesPlugin.java:463)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:771)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:764)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:721)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:941)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:318)
	at org.eclipse.osgi.container.Module.doStart(Module.java:571)
	at org.eclipse.osgi.container.Module.start(Module.java:439)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:454)
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:107)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:531)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:324)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:327)
	at org.eclipse.osgi.internal.loader.sources.SingleSourcePackage.loadClass(SingleSourcePackage.java:36)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:398)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:352)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:344)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:160)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:140)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:380)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:235)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:669)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:608)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1515)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1488)
!SUBENTRY 2 org.eclipse.core.resources 4 567 2024-09-19 15:43:36.828
!MESSAGE The project description file (.project) for 'lab2' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
!SUBENTRY 1 org.eclipse.core.resources 4 567 2024-09-19 15:43:36.828
!MESSAGE Could not read metadata for 'lab2_bsp'.
!STACK 1
org.eclipse.core.internal.resources.ResourceException: The project description file (.project) for 'lab2_bsp' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.read(FileSystemResourceManager.java:888)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:902)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:882)
	at org.eclipse.core.internal.resources.SaveManager.restore(SaveManager.java:733)
	at org.eclipse.core.internal.resources.SaveManager.startup(SaveManager.java:1588)
	at org.eclipse.core.internal.resources.Workspace.startup(Workspace.java:2386)
	at org.eclipse.core.internal.resources.Workspace.open(Workspace.java:2157)
	at org.eclipse.core.resources.ResourcesPlugin.start(ResourcesPlugin.java:463)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:771)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:764)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:721)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:941)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:318)
	at org.eclipse.osgi.container.Module.doStart(Module.java:571)
	at org.eclipse.osgi.container.Module.start(Module.java:439)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:454)
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:107)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:531)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:324)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:327)
	at org.eclipse.osgi.internal.loader.sources.SingleSourcePackage.loadClass(SingleSourcePackage.java:36)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:398)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:352)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:344)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:160)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:140)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:380)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:235)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:669)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:608)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1515)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1488)
!SUBENTRY 2 org.eclipse.core.resources 4 567 2024-09-19 15:43:36.828
!MESSAGE The project description file (.project) for 'lab2_bsp' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-09-19 15:43:42.360
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more

!ENTRY org.eclipse.core.resources 4 567 2024-09-19 15:43:44.785
!MESSAGE The project description file (.project) for 'lab2' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.

!ENTRY org.eclipse.core.resources 4 567 2024-09-19 15:45:13.791
!MESSAGE The project description file (.project) for 'lab2' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.

!ENTRY org.eclipse.core.resources 4 568 2024-09-19 15:48:41.765
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.core.resources 4 568 2024-09-19 15:58:24.123
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.core.resources 4 568 2024-09-19 15:58:29.928
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.core.resources 4 568 2024-09-19 15:58:33.904
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.core.resources 4 568 2024-09-19 15:58:38.141
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.core.resources 4 568 2024-09-19 15:58:48.520
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.core 1 0 2024-09-19 15:58:51.687
!MESSAGE Indexed 'hello_lab1_bsp' (89 sources, 158 headers) in 1.5 sec: 5,541 declarations; 9,661 references; 0 unresolved inclusions; 2 syntax errors; 8 unresolved names (0.053%)

!ENTRY org.eclipse.cdt.core 1 0 2024-09-19 15:58:51.802
!MESSAGE Indexed 'lab2' (2 sources, 35 headers) in 0.112 sec: 1,132 declarations; 2,068 references; 0 unresolved inclusions; 0 syntax errors; 0 unresolved names (0%)

!ENTRY org.eclipse.cdt.core 1 0 2024-09-19 15:58:52.497
!MESSAGE Indexed 'lab2_bsp' (89 sources, 157 headers) in 0.663 sec: 5,524 declarations; 9,695 references; 0 unresolved inclusions; 3 syntax errors; 8 unresolved names (0.053%)

!ENTRY org.eclipse.cdt.core 1 0 2024-09-19 15:58:52.548
!MESSAGE Indexed 'hello_lab1' (1 sources, 26 headers) in 0.048 sec: 995 declarations; 1,660 references; 0 unresolved inclusions; 0 syntax errors; 0 unresolved names (0%)

!ENTRY org.eclipse.core.resources 4 568 2024-09-19 16:03:49.808
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.ui.ide 4 4 2024-09-20 22:47:14.657
!MESSAGE Problems saving workspace

!ENTRY org.eclipse.ui.ide 4 1 2024-09-20 22:47:14.663
!MESSAGE Problems occurred while trying to save the state of the workbench.
!SUBENTRY 1 org.eclipse.core.resources 4 568 2024-09-20 22:47:14.663
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeFileOutputStream.<init>(SafeFileOutputStream.java:51)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSave(SaveManager.java:1658)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSave(SaveManager.java:1740)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1197)
	at org.eclipse.core.internal.resources.Workspace.save(Workspace.java:2270)
	at org.eclipse.ui.internal.ide.application.IDEWorkbenchAdvisor$4.run(IDEWorkbenchAdvisor.java:453)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)
!SESSION 2024-09-24 11:06:36.921 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_NZ
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/intel/fpga/18.10/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/intel/fpga/18.10/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-09-24 11:07:01.165
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more

!ENTRY org.eclipse.core.resources 4 568 2024-09-24 11:10:20.912
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.core 1 0 2024-09-24 11:10:22.964
!MESSAGE Indexed 'pacemaker_bsp' (89 sources, 68 headers) in 1.81 sec: 1,444 declarations; 3,713 references; 125 unresolved inclusions; 6 syntax errors; 212 unresolved names (3.9%)

!ENTRY org.eclipse.core.resources 4 568 2024-09-24 11:10:24.755
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.core 1 0 2024-09-24 11:10:30.030
!MESSAGE Indexed 'pacemaker' (1 sources, 21 headers) in 0.059 sec: 983 declarations; 1,599 references; 0 unresolved inclusions; 0 syntax errors; 0 unresolved names (0%)

!ENTRY org.eclipse.core.resources 4 568 2024-09-24 11:15:24.803
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.core.resources 4 568 2024-09-24 11:22:39.404
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.core.resources 4 568 2024-09-24 11:28:07.470
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.core.resources 4 568 2024-09-24 14:10:31.155
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers.snap (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:45)
	at org.eclipse.core.internal.localstore.SafeChunkyOutputStream.<init>(SafeChunkyOutputStream.java:41)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1769)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSnap(SaveManager.java:1853)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1222)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1163)
	at org.eclipse.core.internal.resources.DelayedSnapshotJob.run(DelayedSnapshotJob.java:52)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.ui.ide 4 4 2024-09-24 20:57:18.679
!MESSAGE Problems saving workspace

!ENTRY org.eclipse.ui.ide 4 1 2024-09-24 20:57:18.691
!MESSAGE Problems occurred while trying to save the state of the workbench.
!SUBENTRY 1 org.eclipse.core.resources 4 568 2024-09-24 20:57:18.691
!MESSAGE Could not write metadata for '/RemoteSystemsTempFiles'.
!STACK 0
java.io.FileNotFoundException: C:\Users\juane\Desktop\University\Part 3\303\303-Pacemaker\pacemaker\labs\.metadata\.plugins\org.eclipse.core.resources\.projects\RemoteSystemsTempFiles\.markers (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.eclipse.core.internal.localstore.SafeFileOutputStream.<init>(SafeFileOutputStream.java:51)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSave(SaveManager.java:1658)
	at org.eclipse.core.internal.resources.SaveManager.visitAndSave(SaveManager.java:1740)
	at org.eclipse.core.internal.resources.SaveManager.save(SaveManager.java:1197)
	at org.eclipse.core.internal.resources.Workspace.save(Workspace.java:2270)
	at org.eclipse.ui.internal.ide.application.IDEWorkbenchAdvisor$4.run(IDEWorkbenchAdvisor.java:453)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)
!SESSION 2024-09-26 14:08:49.091 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_NZ
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/intel/fpga/18.10/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization D:/intel/fpga/18.10/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-09-26 14:09:22.127
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more
